src/net.h

```diff
@@ -954,6 +961,16 @@ class CNode
         assert(false);
     }
 
+    /**
+     * Get network the peer connected through.
+     *
+     * CNetAddr::GetNetClass() is used excluding inbound connections that are
+     * explicitly tagged as connections from the Tor onion service(s).
```

Comment:
It's for sure that my comment could be improved, but I don't think that it is a correct assumption that it's responsibility of `CNetAddr` class to detect the network a peer is connected through. I mean "CNetAddr::GetNetClass() does not currently detect" part.

Reply:
Yes. That seems like a worthwhile clarification. Maybe:

`Returns Network::NET_ONION for *inbound* onion connections, and CNetAddr::GetNetClass() otherwise. The latter cannot be used directly because it's not the responsability of the CNetAddr class to know the actual network a peer is connected through.`

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/19998#issuecomment-703097616).